SCRD-6374 Remove translation for logging message

guglo opened the pull request:
The commit includes:
1. Removed translation of the logging messages in replace server
   process
2. Updated logMessage to append new line '\n' or no new line
   based on passed in  params (SCRD-6375)
3. Removed unused messages.
4. Allowed error message from action run to show up in the page banner

Commit:
SCRD-6374 Remove translation for logging message

The commit includes:
1. Removed translation of the logging messages in replace server
   process
2. Updated logMessage to append new line '\n' or no new line
   based on passed in  params (SCRD-6375)
3. Removed unused messages.
4. Allowed error message from action run to show up in the page banner

guglo commented:
Ethan has a good idea to allow translation only to English. Will try his idea instead.

jeremy-moffitt reviewed (approved):
LGTM

GarySmith commented on src/components/PlaybookProgress.js:
```diff
@@ -781,11 +781,20 @@ class PlaybookProgress extends Component {
   }
 
-  logMessage = (message) => {
+  logMessage = (message, notAppendNewLine) => {
```
`notAppendNewLine` is confusing. Perhaps we can change the signature to the positive case and default it to `true`, like:
```
logMessage = (message, addNewlineIfMissing=True) 
```

guglo replied:
@GarySmith  that is better. will update

GarySmith commented on src/components/PlaybookProgress.js:
```diff
@@ -781,11 +781,20 @@ class PlaybookProgress extends Component {
+    if (!notAppendNewLine) {
+      var hasNewLine = /\r|\n/.exec(message);
+      if (!hasNewLine) {
+        message = '\n' + message + '\n';
```
I don't think we should have a newline before the message, or else all of our messages will be double-spaced.

guglo replied:
@GarySmith ok will remove it

Commit:
Addressed comments

1. Have a better name of param in logMessage
2. Removed extra line break in logMessage
3. Added extra line break in logProgressResponse and logProgressError

Commit:
Merge branch 'master' into rmtrans2
